X16 Reference - 12 - IO Programming.md

```diff
@@ -51,7 +51,7 @@ the PS/2 keyboard and mouse.
 | $01      | $01           | Hard reboot               |
 | $02      | $00           | Inject RESET              |
 | $03      | $00           | Inject NMI                |
-| $05      | $00..$FF     | Activity LED brightness   |
+| $05      | $00/$01     | Activity LED off/on   |
```

Comment:
Let's use $00/$FF.  On my system $01 is still off.  It doesn't change until it crosses $80.

Reply:
I have created a pull request for SMC as well that will follow this. Any value above 0 will turn the LED on.

Reply:
That isn't what kernal code does at the moment.  It uses $00/$FF.  I see no reason to change this.

Reply:
How about $00/$01-$FF = Activity LED OFF/ON ? That way it will follow the SMC change I made and leave no values undefined.

Reply:
Also, documenting that $01 is an on-value would not be true for all of the SMC versions that are out in the field.  If people relied on this, old SMCs would not light up, and there's no reason for it to be that way.

I would recommend having the docs reflect values which are true for as many cases as possible.  This is regardless of any changes to the smc firmware.  Perhaps a future SMC could support PWM, and in that case $00 would be off and $FF would be on, just as it is true today

Reply:
I see what you mean. Do we have any kind of version information in the docs? It seems it would be nice to know which versions of ROM, SMC & VERA are documented.

Reply:
I don't think we do.  Particularly since R43 had a breaking SMC change, this is probably our starting point.

Reply:
I create a tag on the x16-docs repo whenever a ROM release is done. This is about the only way we've got any sort of doc versioning that applies to a specific ROM version.  The x16-docs repo otherwise tends to reflect the state of ROM/emu in the master branch.